client/src/desktop/components/AdventurerStats.tsx

```
@@ -279,18 +275,41 @@ export default function AdventurerStats() {
+                  display: 'inline-flex',
+                  alignItems: 'center',
+                  color: selectedStats[stat as keyof typeof STAT_DESCRIPTIONS] > 0 ? '#4caf50' : '#d0c98d',
+                  ...(statChanges[stat as keyof typeof STAT_DESCRIPTIONS] === 'increase' && {
+                    animation: `${statIncrease} 1.5s ease-out`,
+                  }),
+                  ...(statChanges[stat as keyof typeof STAT_DESCRIPTIONS] === 'decrease' && {
+                    animation: `${statDecrease} 1.5s ease-out`,
+                  }),
+                }}
+              >
+                {adventurer?.stats?.[stat as keyof typeof STAT_DESCRIPTIONS]! + selectedStats[stat as keyof typeof STAT_DESCRIPTIONS]!}
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)